README.md

```diff
@@ -1033,21 +506,12 @@ may get redirected for unknown hosts to 92.242.140.21.  To test this run
 nslookup somehost
 ```
 
-If you have this situation, then it is recommended to Google's DNS servers.  [Here](https://developers.google.com/speed/public-dns/docs/using) 
-are some instructions from Google.
-
-### Can't run tests in Eclipse
```

Comment:
This section and the following section don't seem helpful and almost seem condescending